examples/tutorial.py

```diff
@@ -929,13 +1038,402 @@ def get_overlapping_labels(array1, array2):
 
 # %%
 figure, axes = plt.subplots(1, 2)
-axes[0].imshow(zarr.open('sample_data.zarr', 'r')['smoothed_for_seg'][:].transpose(1,2,0), origin="lower")
-axes[1].imshow(zarr.open('sample_data.zarr', 'r')['blue_objects_with_context'][:], cmap=colormap, interpolation="nearest", origin="lower")
+axes[0].imshow(
+    zarr.open("sample_data.zarr", "r")["smoothed_for_seg"][:].transpose(1, 2, 0),
+    origin="lower",
+)
+axes[1].imshow(
+    zarr.open("sample_data.zarr", "r")["blue_objects_with_context"][:],
+    cmap=colormap,
+    interpolation="nearest",
+    origin="lower",
+)
 
 # %% [markdown]
 # All the labels are now consistent! If you re-run the previous cell with `read_write_conflict=False`, you should see an inconsistent result again due to the race conditions, even though the process function still reads the neighboring output.
 
 # %% [markdown]
 # **IMPORTANT PERFORMANCE NOTE:** Be aware that `read_write_conflicts` is set to `True` by default and can lead to performance hits in cases where you don't need it, so be sure to turn it off if you want every block to be run in parallel!
 
+# %% [markdown]
```

Comment:
This still doesn't really make sense as one big tutorial. I think you should put this part that you wrote first as a "Quick start", and then have the part I wrote below as an "In-Depth Tutorial."